exercises/practice/isogram/.docs/instructions.md

```diff
@@ -0,0 +1,14 @@
+# Instructions
+
+Determine if a word or phrase is an isogram.
```

Comment:
I am presuming this is coming from problem-specifications... otherwise I would suggest including sentence as a size indicator here not only as long as a phrase.

Reply:
Yeah, that's from the problem-specifications repository. The test suite doesn't specifically use sentences, and a paragraph is just as valid of an input as a sentence or word. I think it'd be best to say a sequence of characters without hinting at length, but adding that at the end as an append feels clunky.

Reply:
Leave it, no need to change it.  In reality a sentence that exceeds the character length of the alphabet is the limit of possibility for an isogram anyway.